geth/node/rpc_test.go

```diff
@@ -20,9 +33,70 @@ type RPCTestSuite struct {
 
 func (s *RPCTestSuite) SetupTest() {
 	require := s.Require()
-	s.NodeManager = node.NewNodeManager()
+
+	s.NodeManager = proxy.NewRPCRouter(node.NewNodeManager())
+	require.NotNil(s.NodeManager)
+	require.IsType(&proxy.RPCRouter{}, s.NodeManager)
+}
+
+func (s *RPCTestSuite) TestUpstreamCallRPC() {
```

Comment:
TestRPCClientConnection was to validate that we are able to hit an exterior rpc server with requests and receive response. I need to validate the expected message format for request and response received and that this matched the needed requirements as to an upstream node. 

But as we had validated the behaviour of sendTransaction, this can be removed.